README.md

```diff
@@ -15,21 +15,20 @@ NPM 2.x or greater and Node.js 4.x.
 4. **apigee_analytics_client_id** please submit a ticket with Apigee Support to obtain apigee_analytics_client_id and apigee_analytics_secret
 5. **apigee_analytics_secret**
 
-##### Installation using NPM
-Unzip `apigee-analytics-forwarder_*.zip` file, cd into the folder where it was expanded and install with NPM:
```

Comment:
I'll modify this step to do installation from NPM directly as `$ npm install apigee-analytics-collector -g` instead of cloning the repo.